src/PureCloud.Client/Apis/WorkforceManagementApi.cs

```diff
@@ -0,0 +1,396 @@
+using System.Collections.Specialized;
+using System.Net.Http.Json;
+using System.Text.Json;
+using Microsoft.Extensions.Options;
+using PureCloud.Client.Contracts;
+using PureCloud.Client.Http;
+using PureCloud.Client.Json;
+using PureCloud.Client.Models;
+
+namespace PureCloud.Client.Apis;
+
+/// <inheritdoc />
+public sealed class WorkforceManagementApi : IWorkforceManagementApi
+{
+    private readonly HttpClient _httpClient;
+    private readonly JsonSerializerOptions _options;
+
+    public WorkforceManagementApi(IHttpClientFactory httpClientFactory, IOptions<PureCloudJsonSerializerOptions> options)
+    {
+        _httpClient = httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
+        _options = options.Value.JsonSerializerOptions;
+    }
+
+    /// <inheritdoc />
+    public async Task<BusinessUnitListing> GetBusinessUnitsAsync(string feature = null, string divisionId = null, CancellationToken cancellationToken = default)
+    {
+        var parameters = new NameValueCollection();
+
+        if (feature != null)
+        {
+            parameters.Add("feature", UriHelper.ParameterToString(feature));
+        }
+
+        if (divisionId != null)
+        {
+            parameters.Add("divisionId", UriHelper.ParameterToString(divisionId));
+        }
+
+        var uri = UriHelper.GetUri("api/v2/workforcemanagement/businessunits", parameters);
+
+        var response = await _httpClient.GetAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadFromJsonAsync<BusinessUnitListing>(_options, cancellationToken);
+    }
+
+    /// <inheritdoc />
```

Comment:
Expands should be IEnumerable and it should be renamed to expands.  In the foreach loop it would be expand in expands